packages/cozy-harvest-lib/src/components/InAppBrowser.jsx

```diff
@@ -56,7 +56,15 @@ const InAppBrowser = ({ url, onClose, intentsApi = {} }) => {
     return function cleanup() {
       closeInAppBrowser()
     }
-  }, [isReady, url, onClose])
+  }, [
+    isReady,
+    url,
+    onClose,
+    closeInAppBrowser,
+    fetchSessionCode,
+    showInAppBrowser,
+    tokenParamName
+  ])
```

Comment:
are you sure of this change? have you verified that don't break nothing?

to make a safer change, I would have disabled eslint here.